src/app/service/ndkprovider.service.ts

```diff
@@ -395,6 +391,38 @@ export class NdkproviderService {
     this.followedTopicsEmitter.emit(followedTopicsToPublish);
   }
 
+  async followUnfollowContact(hexPubKeyToFollow:string, follow:boolean){
+    var contactListFromCache = (await this.dbService.peopleIFollow.toArray()).map(item=>item.hexPubKey);
+
+    // if it is a follow event, attach the new contact to the list or else remove from the list if found and publish
+    if(follow){
+      contactListFromCache.push(hexPubKeyToFollow);
+    }else{
+      const index = contactListFromCache.indexOf(hexPubKeyToFollow);
+
+      if (index > -1) {
+        contactListFromCache.splice(index, 1);
+      }
+    }
+
+    var tags:NDKTag[] = contactListFromCache.map((item, index)=>{
+      return ["p", item.toString()]
+    });
+
+    const ndkEvent = new NDKEvent(this.ndk);
+    if(this.currentUser){
+      ndkEvent.pubkey = this.currentUser?.hexpubkey()
+    }
+
+    ndkEvent.tags = tags;
+    ndkEvent.kind = 3;
+    ndkEvent.content = "{\"wss://nos.lol\":{\"read\":true,\"write\":true}, \"wss://relay.nostr.band\":{\"read\":true,\"write\":true},\"wss://relay.f7z.io\":{\"read\":true,\"write\":true},\"wss://relay.damus.io\":{\"read\":true,\"write\":true},\"wss://nostr.mom\":{\"read\":true,\"write\":true}, \"wss://no.str.cr\":{\"read\":true,\"write\":true}}";
```

Comment:
Any specific reason we are populating `content` with relays?

As per [NIP-02](https://github.com/nostr-protocol/nips/blob/master/02.md) , 

> `content` can be anything and should be ignored